content/_code-samples/build-a-wallet/js/1_hello.js

```diff
@@ -0,0 +1,71 @@
+const { app, BrowserWindow } = require('electron')
+
+const path = require('path')
+const xrpl = require("xrpl")
+
+const testnetUrl = "wss://s.altnet.rippletest.net:51233"
+
+/**
+ * This function creates a WebService client, which connects to the XRPL and fetches the latest ledger index.
+ *
+ * @returns {Promise<number>}
+ */
+const getValidatedLedgerIndex = async () => {
+    const client = new xrpl.Client(testnetUrl)
+
+    await client.connect()
+
+    const ledgerRequest = {
```

Comment:
It would be useful to include links to the relevant docs when using the ledger (Ex. Here a doc to the ["ledger"](https://xrpl.org/ledger.html#ledger) page would be helpful context)

Reply:
Done